idb/postgres/postgres_integration_test.go

```diff
@@ -172,24 +167,14 @@ func TestAssetCloseReopenTransfer(t *testing.T) {
 	///////////
 	// Given // A round scenario requiring subround accounting: AccountA is funded, closed, opts back, and funded again.
 	///////////
-	createAsset := test.MakeAssetConfigTxn(0, total, uint64(6), false, "mcn", "my coin", "http://antarctica.com", test.AccountD)
-	optInA1 := test.MakeAssetOptInTxn(assetid, test.AccountA)
-	optInA2 := test.MakeAssetOptInTxn(assetid, test.AccountA)
-	fundA := test.MakeAssetTransferTxn(assetid, amt, test.AccountD, test.AccountA, basics.Address{})
-	optInB := test.MakeAssetOptInTxn(assetid, test.AccountB)
-	optInC := test.MakeAssetOptInTxn(assetid, test.AccountC)
-	closeA := test.MakeAssetTransferTxn(assetid, 1000, test.AccountA, test.AccountB, test.AccountC)
-	payMain := test.MakeAssetTransferTxn(assetid, amt, test.AccountD, test.AccountA, basics.Address{})
```

Comment:
I'm curious what others thing here.

Should we keep this code commented out to serve as documentation for whats inside the validated blocks?

Reply:
Yeah I think it's fine to keep these as comments for now. Will they be restored once we fully migrate?

Reply:
I don't think we'll restore these until we have a way to create the state deltas for tests.